EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/80190
+oovoo.com#?#.elementor-element > .elementor-widget-container > center > p:contains(/^Advertisement$/):upward(2)

```

Comment:
just `center > p[style=" font-size: smalll; "]` is safe here

Reply:
Fixed.

Reply:
@Alex-302